File: PB WEB/ejercicio5.java
public class Histograma {
    public static void main(String[] args) {
        int[] myArray = {1, 2, 3, 4, 5, 2, 3, 4, 1, 3};
        int[] frequency = new int[6]; // 6 porque los números están del 1 al 5, y se necesita una posición más para la frecuencia de los números no presentes
        
        for (int i = 0; i < myArray.length; i++) {
            frequency[myArray[i]]++;
        }
        
        for (int i = 1; i < frequency.length; i++) {
            System.out.print(i + ": ");
            for (int j = 0; j < frequency[i]; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: PB WEB/ejercicio5.js
const myArray = [1, 2, 2, 3, 4, 4, 4, 5, 5, 5];
const frequency = new Array(6).fill(0); 


for (let i = 0; i < myArray.length; i++) {
  const num = myArray[i];
  frequency[num]++;
}


console.log("Histograma:");
for (let i = 1; i <= 5; i++) {
  let output = i + " | ";
  for (let j = 0; j < frequency[i]; j++) {
    output += "*";
  }
  output += " (" + frequency[i] + ")";
  console.log(output);
}